app/core/Tracker2/BaseTracker.js

```diff
@@ -33,6 +29,15 @@ export default class BaseTracker extends EventEmitter {
 
   async trackTiming (duration, category, variable, label) {}
 
+  watchForCookieConsentChanges (store) {
+    this.cookieConsentDeclined = store.getters['tracker/cookieConsentDeclined']
+
+    store.watch(
+      (state, getters) => getters['tracker/cookieConsentDeclined'],
+      (result) => { this.cookieConsentDeclined = result }
```

Comment:
This is really cool!

Reply:
Yeah!  This is actually what is happening when you call `mapGetters` / `mapState` / etc, those functions just make it easier to bind these calls to vue props!